### DISCUSSION TITLE
Quais as diferenças entre Angular e Handlebars

### DISCUSSION THREAD

**Discussion by peidrao** [frontendbr/forum#1714] [Perguntas e Respostas]
Então pessoal, até algum dia atrás pensava que **Handlebars** era uma versão do **Angular** com menos funcionalidades, porém, descobri que não é assim. Gostaria de saber por parte de vocês as diferenças entre uma e outra e mesmo sendo diferentes, existe alguma coisa que uma tem e outra não e vice-versa. Esse assunto me deixou bastante curioso, desde já muito obrigado!

**Comment by LuanDc** [comment]
Handlebars é um template engine do nodeJS. Diferente do angular, ele cria páginas HTML no lado do servidor e o seu navegador ler o conteúdo HTML gerado dinamicamente pelo Handlebars. O Angular é diferente, ele cria uma aplicação que roda no lado do cliente, manipulando DOM em tempo de execução, ou seja, você não precisa da refresh na página a cada requisição ao navegador.

**Comment by LuanDc** [comment]
Pra entender melhor o angular é importante ter em mente conceitos como requisição AJAX e manipulação de DOM.

**Comment by peidrao** [comment]
Poderia me fornecer alguma fonte para estudar sobre isso?

**Comment by LuanDc** [comment]
Procura algumas fontes de informações sobre sistemas SPA e sistemas monolíticos. Alguns artigos na Medium são interessantes.

**Comment by peidrao** [comment]
Certo, irei procurar sim! **Muito obrigado**
